packages/wallets/src/features/cfd/screens/CompareAccounts/InstrumentsLabelHighlighted.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { getHighlightedIconLabel } from './compareAccountsConfig';
+import { CFD_PLATFORMS, MARKET_TYPE } from './constants';
+import InstrumentsIconWithLabel from './InstrumentsIconWithLabel';
+import './InstrumentsLabelHighlighted.scss';
+
+type TInstrumentsLabelHighlighted = {
+    isDemo: boolean; // TODO: remove this prop
```

Comment:
there is comment to remove it, why not remove it 😸

Reply:
Updated.